Header header
string state
string command
float32 current_speed